Source/CarthageKit/Version.swift

```diff
@@ -83,6 +80,23 @@ public struct SemanticVersion: VersionType {
 		                       patch: self.patch,
 		                       preRelease: self.preRelease)
 	}
+	
+	/// Set of valid characters for SemVer major.minor.patch section
+	fileprivate static let versionCharacterSet = CharacterSet(charactersIn: ".")
+		.union(CharacterSet.decimalDigits)
```

Comment:
Our fault for not documenting this to begin with, but `CharacterSet.decimalDigits` was avoided because of the weirdness that `CharacterSet.decimalDigits.contains("\u{FF13}") == true` for full-width scalars like [`FULLWIDTH DIGIT THREE`](https://www.unicode.org/charts/nameslist/n_FF00.html#FF13)

Reply:
No problem, I can change it back to numbers and add a comment to avoid this in future.

Reply:
Addressed here: https://github.com/Carthage/Carthage/pull/2458